src/current/cockroachcloud/billing-management.md

```diff
@@ -51,6 +62,30 @@ If you used a CockroachDB {{ site.data.products.dedicated }} trial code while [c
 Your credit card will be charged after the trial ends.
 {{site.data.alerts.end}}
 
+</section>
+
+<section class="filter-content" markdown="1" data-scope="marketplace">
+
+## Subscribe through AWS Marketplace
+
+To subscribe to CockroachDB {{ site.data.products.cloud }} through the AWS Marketplace:
+
+1. Navigate to the [CockroachDB product page](https://aws.amazon.com/marketplace/pp/prodview-3zbkzekdohwly) on the AWS Marketplace and click **View purchase options**.
```

Comment:
The link to the new AWS listing is https://aws.amazon.com/marketplace/pp/prodview-n3xpypxea63du